mypyc/primitives/int_ops.py

```diff
@@ -129,6 +130,12 @@ def int_compare_op(name: str, c_function_name: str) -> None:
     format_str='{dest} = {args[0]} + {args[1]} :: short_int',
     emit=simple_emit('{dest} = {args[0]} + {args[1]};'))
 
+int_equal = c_custom_op(
```

Comment:
Add comment. Maybe rename this to something like `int_equal_` to highlight that this is just a part of an implementation.